adapter/ipListChecker/ipListChecker_test.go

```diff
@@ -109,14 +107,33 @@ func TestBasic(t *testing.T) {
 	lc := a.(*listChecker)
 
 	// do a NOP refresh of the same data
-	lc.refreshList()
+	if err := lc.fetchList(); err != nil {
+		t.Errorf("Unable to refresh list: %v", err)
+	}
 
-	// now try to parse the bad list
-	useGoodList = false
-	lc.refreshList()
-	list := lc.getList()
-	if len(list.payload) != 2 {
-		t.Errorf("Expecting %d, got %d entries", len(badList.WhiteList)-1, len(list.payload))
+	// now try to parse a list with errors
+	listToUse = 1
+	if err := lc.fetchList(); err != nil {
+		t.Errorf("Unable to refresh list: %v", err)
+	}
+	list := lc.getListState()
+	if len(list.entries) != 2 {
+		t.Errorf("Expecting %d, got %d entries", len(list1.WhiteList)-1, len(list.entries))
```

Comment:
I definitely disagree with the importance of such formatting conventions. It really adds nothing to the debugging of the test failure. It's not easier to read, not easier to understand, and above all, it will be seen by probably no one.

It's interesting to note that the particular bit of advice about useful test failures in the code review doc immediately precedes the worse piece of style advice I've ever seen, namely "Variable names in Go should be short rather than long". I point this out to highlight the fact it's OK to ignore suggestions that don't help.